docs/content/docs/guide/gitops_commands.md

```diff
@@ -29,6 +29,12 @@ Roses are red, violets are blue. Pipelines are bound to flake by design.
 /test <pipelinerun-name>
 ```
 
+{{< hint info >}}
+
+Please be aware that GitOps commands such as `/test` and others will not function on closed Pull Requests or Merge Requests.  
+
```

Comment:
can you add the supported tag version  like here https://pipelinesascode.com/docs/guide/matchingevents/#matching-a-pipelinerun-on-a-regexp-in-a-comment